tools/libxl/libxl_arm.c

```diff
@@ -952,7 +985,7 @@ static int libxl__prepare_dtb(libxl__gc *gc, libxl_domain_build_info *info,
         FDT( make_hypervisor_node(gc, fdt, vers) );
 
         if (pfdt) {
-            FDT( copy_partial_fdt(gc, fdt, pfdt) );
+            FDT( copy_partial_fdt(gc, fdt, pfdt, info) );
             FDT( copy_coprocs_nodes(gc, fdt, pfdt, info) );
```

Comment:
These patches are based on our xt-master-wip that has already contain the coprocs patches too. Although the coproc and ipmmu seems don't intersect with each other critically,  I can rebase on xen.org.